submissions/IRETIOGEDENGBE/CSS/Homepage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="homepage.css" rel="stylesheet"
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Homepage</title>
+</head>
+<body>
+    <header>   
+        <h1>Aloha Store</h1>
+            <div class="dropdown-ctn">

```

Comment:
Indentation is off here... See [Styleguide.md](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md)